numpy/__init__.pyi

```diff
@@ -389,7 +389,6 @@ from numpy._core.numeric import (
 from numpy._core.numerictypes import (
     isdtype as isdtype,
     issubdtype as issubdtype,
-    cast as cast,
```

Comment:
This one doesn't exist at all, so should be removed.

Reply:
It already was :)

Reply:
Ah OK, didn't realizes stubs are happy to import when things don't exist.